CMakeLists.txt

```diff
@@ -233,6 +233,9 @@ endif()
 target_link_libraries(openomf ${CORELIBS})
 set_property(TARGET openomf PROPERTY C_STANDARD 11)
 
+# This is to make loading test resources possible
+add_compile_definitions(OPENOMF_ROOT_DIR="${CMAKE_SOURCE_DIR}")
```

Comment:
Probably best move this to the "testing stuff" block -- afaiks not something that should be needed for the main binary.

Reply:
Moved it to the testing stuff block and renamed it TESTS_ROOT_DIR